src/util/fs.ts

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const papaParser = require('papaparse');
+
+export function readFile(csvPath: string) {
```

Comment:
please rename this function's name. Something like `parseCSVToJSON` makes more sense

Reply:
updated